### ISSUE TITLE
don't use mean scale and scale offset

### ISSUE THREAD

**Issue by huiyang0613** [goldoak/DMSR#3]
![image](https://github.com/goldoak/DMSR/assets/115716973/af116312-65bd-45cd-9f29-16c3a217584c)

nice work~
If you don't use mean scale and scale offset, what are the metrics in the table?

**Comment by goldoak** [comment]
Sorry, I don't fully understand your question. The same metrics are used throughout all experiments. If you don't use the mean scale, it will severely affect the translation estimation since we only provide a relative depth map in the input.

**Comment by huiyang0613** [comment]
> Sorry, I don't fully understand your question. The same metrics are used throughout all experiments. If you don't use the mean scale, it will severely affect the translation estimation since we only provide a relative depth map in the input.

Sorry I didn't make it clear, I mean what is the value of IOU and 10°, 10cm if you don't use mean scale

**Comment by goldoak** [comment]
We have done that experiment but didn't report that in the paper due to the page limit. The conclusion is that without mean scales, both translation and scale estimation will be affected severely. The rotation accuracy will also drop a little bit.

**Comment by goldoak** [comment]
I am preparing for the full dataset and training code and probably can open-source them this weekend. You can also test it yourself by the time.

**Comment by huiyang0613** [comment]
> I am preparing for the full dataset and training code and probably can open-source them this weekend. You can also test it yourself by the time.

Thank you very much for your reply ~ I will try it ~
